tests/tests_animation/test_anim.py

```diff
@@ -0,0 +1,11 @@
+import matplotlib
+
+from poliastro.examples import iss
+from poliastro.plotting.static import StaticOrbitPlotter
+
+
+def test_type_of_anim():
```

Comment:
We need to figure out how to test animations. I wonder if there exists something like the matplotlib-mpl pytest extension 🤔

Reply:
with my search and all, I feel that there is no plugin for testing animation video or gif. 
we can design our own tester for animation but efficiency is the concern